api/subject_query.js

```diff
@@ -0,0 +1,49 @@
+    const grad_url = 'https://api.metalab.csun.edu/curriculum/api/2.0/plans/graduate';
+    const courses_url = 'https://api.metalab.csun.edu/curriculum/api/2.0/courses';
+
+    var subject_to_search;
+    var subjectArray = [
+        "A/R","AAS","ACCT","AE","AFRS","AIS","AM","ANTH","ARAB","ARMN","ART","ASTR",
+        "ATHL","BIOL","BLAW","BUS","CADV","CAS","CCE","CD","CE","CECS","CHEM","CHIN",
+        "CHS","CIT","CJS","CLAS","CM","COMP","COMS","CTVA","DEAF","ECE","ECON","EDUC",
+        "EED","ELPS","ENGL","EOH","EPC","FCS","FIN","FLIT","FREN","GBUS","GEOG","GEOL",
+        "GWS","HEBR","HHD","HIST","HSCI","HUM","IS","ITAL","JAPN","JOUR","JS","KIN",
+        "KOR","LING","LRS","MATH","MCOM","ME","MGT","MKT","MPA","MSE","MUS","NURS",
+        "PERS","PHIL","PHSC","PHYS","POLS","PSY","PT","QS","RE","RS","RTM","RUSS","SBS",
+        "SCI","SCM","SED","SOC","SOM","SPAN","SPED","SUST","SWRK","TH","UNIV","URBS"
+    ];
+
+    function searchSubject(){
```

Comment:
allow the `searchSubject` option to take in a parameter. 
ex: 
`searchSubject(subject)` where `subject=comp` or a different major